lib/ansible/module_utils/network/common/utils.py

```diff
@@ -444,104 +445,9 @@ def contains_vars(self, data):
         return False
 
 
-def is_netmask(val):
-    parts = str(val).split('.')
-    if not len(parts) == 4:
-        return False
-    for part in parts:
-        try:
-            if int(part) not in VALID_MASKS:
-                raise ValueError
-        except ValueError:
-            return False
-    return True
-
-
-def is_masklen(val):
-    try:
-        return 0 <= int(val) <= 32
-    except ValueError:
-        return False
-
-
 def to_bits(val):
```

Comment:
Although only used by the network modules, it isn't specific to them, so this should probably move as well.